.github/workflows/rail_inspector.yml

```diff
@@ -0,0 +1,24 @@
+name: Rail Inspector
```

Comment:
I'm wondering if we need to add this workflow if we're going to move this to buildkite soon anyways 🤔